docs/markdown/Reference-tables.md

```diff
@@ -0,0 +1,54 @@
+# Reference tables
+
+## Compiler ids
+
```

Comment:
I would suggest adding a line here describing the context of these values. Namely that they are the possible return values of `compiler.get_id`.